src/pages/fr/getting-started.md

```diff
@@ -2,107 +2,108 @@
+- **UI-agnostic:** Support de React, Preact, Svelte, Vue, Solid, Lit et plus.
 
-### Environnements d'essais en ligne
+<!-- - **`client:visible` component loading:** If your user never sees it, it never loads. -->
+<!-- - **Image optimizations:** Astro's very own `<Image />` component. -->
+<!-- - **TypeScript support**  -->
+<!-- - **File-based routing:** Every file in the pages directory becomes a route. -->
 
-Visitez [astro.new](https://astro.new/) pour la façon la plus simple "d'essayer avant d'acheter". Choisissez parmi une variété de Templates de démarrage et commencez à construire une version complète et fonctionnelle d'Astro dans votre navigateur !
+Consultez l'article détaillé [Pourquoi Astro](/fr/concepts/why-astro/) pour en apprendre plus sur ce qui rend Astro spécial. ✨
 
-Ou alors, **lancez instantanément notre projet de démarrage basique** d'un simple clic de bouton :
+## Essayez Astro dans votre navigateur
+
+Visitez [astro.new](https://astro.new/) et choisissez parmi une grandre variété de modèles pour débuter. Testez une version complète et fonctionnelle d'Astro directement dans votre navigateur !
 
 <div style="display: flex; flex-wrap: wrap; gap: 0.5rem;">
-    <Button href="https://astro.new/starter?on=codesandbox">Ouvrir dans CodeSandbox</Button>
-    <Button href="https://astro.new/starter?on=stackblitz">Ouvrir dans StackBlitz</Button>
+  <Button href="https://astro.new/basics?on=stackblitz">Lancer le modèle basique</Button>
+  <Button variant="outline" href="https://astro.new/">Voir tous les modèles →</Button>
 </div>
 
-### Installer Localement
-
-Prêt à installer ?
-
-Créez un nouveau projet prêt localement et en un rien de temps avec notre assistant de création `create-astro` depuis votre terminal de commandes !
-
-```bash
-# Créez un nouveau projet avec NPM
-npm create astro@latest
-
-# ou avec Yarn
-yarn create astro
-
-# ou bien PNPM
-pnpm create astro@latest
-```
-
-⚙️ Notre [Guide d'installation](/fr/install/auto/) contient les instructions complètes et détaillées pour installer Astro avec votre gestionnaire de package favori.
-
-⚙️ Ou alors, jetez un oeil aux instructions pour une [Installation manuelle](/fr/install/manual/).
-
-## Commencez à construire avec Astro
-
-Allez directement à l'essentiel et ajoutez quelques contenus et fonctionnalités à votre site !
-
-🏗️ Ajoutez de nouvelles [Pages Astro](/fr/core-concepts/astro-pages/) et/ou [Pages Markdown](/fr/guides/markdown-content/) à votre site.
-
-🏗️ Créez votre premier [Composant Layout](/fr/core-concepts/layouts/).
-
-🏗️ Ajoutez vos propres [Règles CSS](/fr/guides/styling/) à votre site.
-
-*... et encore plus dans la catégorie **Fonctionnalités***
+## Démarrez votre premier projet
+
+Obtenez un nouveau projet Astro opérationnel localement avec notre assistant utile en ligne de commande `create-astro` !
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # create a new project with npm
```

Comment:
```suggestion
  # créer un nouveau projet avec npm
```

Reply:
I didn't want to translate code comment.

Reply:
Don't you think it would be helpful though?

Reply:
Ok, done.